src/topic_modeling.py

```diff
@@ -98,97 +102,208 @@ def get_bert_embeddings(torch_data):
     return all_embeddings
 
 
-def reduce_dimensions(clustering_data, metric='cosine', n_neighbors=19):
+def get_arguments(default: Dict[str, Dict], input_parameters: Dict):
+    """
+    update default parameters with
+    input parameters
+    :param default: given config
+    :param input_parameters: data to over write
+    :return:
     """
-    Dimensionality reduction for improved clustering.
-    - param clustering_data: input data frame with preprocessed text and encodings
-    - param metric: Dimensions of the output embeddings. Our experience is that dimension of 3 performs well.
-    - param n_neighbors: Number of neighbors
+    for key in input_parameters:
+        if key in default:
+            default.update(
+                {key: input_parameters[key]}
+            )
+
+    return default
+
+
+def normalize_data(clustering_data: List,
```

Comment:
Not sure about this. It was just a one-liner anyways. But I can see the argument that it makes it more modular and allows for later expansion.